physx-sys/build.rs

```diff
@@ -143,8 +143,14 @@ fn main() {
         .define("PX_GENERATE_STATIC_LIBRARIES", "True")
         .define("PX_GENERATE_GPU_PROJECTS", "False")
         .define("TARGET_BUILD_PLATFORM", target_os)
-        .define("CMAKE_C_COMPILER", "clang")
-        .define("CMAKE_CXX_COMPILER", "clang++")
+        .define(
+            "CMAKE_C_COMPILER",
+            env::var("CC").unwrap_or_else(|| "clang".to_owned()),
```

Comment:
This is a result, so you have to do `unwrap_or_else(|_| "clang".to_owned())`. ^ This does not compile